### PR TITLE
Promotional credits

### PR DESCRIPTION
**Pull request by azure562**
- Add promotional credits
- Update comparative reports
- Move reports-related pages into the Reports folder
- Update screenshots
- Update top/bottom results
- General editorial changes

See CMP-4421, CMP-4778

### CODE REVIEW

File: website/docs/cloud-analytics/reports/index.mdx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+## FAQ
+
+### When are the metric amounts (Cost, Usage, etc) for a day counted as "final"?
+
+You can consider numbers "final" for 3 days after the date. With that in mind, there is really no SLA for this and numbers can change any time if there are corrections or delays that needs to be applied.
+
+### Are the reports back-filled/updated, and at what frequency?
+
+Cloud Analytics data and reports are updated hourly.
+
```
Comment (azure562) [inline website/docs/cloud-analytics/reports/index.mdx:80]: @Dror88: the update frequency mentioned here seems to be outdated, can we remove this FAQ entry? Guess it varies by report?

Comment (Dror88) [reply website/docs/cloud-analytics/reports/index.mdx:80]: Why outdated?

Comment (azure562) [reply website/docs/cloud-analytics/reports/index.mdx:80]: it says Cloud Analytics data and reports are `updated hourly`, or report widgets and reports are updated at different frequencies? Report widgets are updated every 12 hours?

**Review by norosa — APPROVED** [review]
LGTM

Nicely done with the nav organization.

My only comment relates to the new screenshots. We decided to stop producing drop shadows in post-production, and instead, apply them with CSS.

You can add a drop shadow, like this:

```mdx
<picture
  img={require('../../assets/png/cmp-settings-auth-saml-values.png')}
  alt="A screenshot of the SAML form field with example information"
  frame="drop-shadow"
/>
```

Note the additional `frame="drop-shadow"` parameter.

You can see this in action on the most recent page I created:

https://help.doit-intl.com/docs/user-management/authentication

Notice how all of the screenshots here use the drop shadow frame, even the closeups of the UI. To make that work, I had to be sure to crop my close-ups so that I wasn't cutting off any visible part of the UI.

I think I might've chosen to omit the drop-shadow for close-ups that crop over visible parts of the UI. But I'm not sure.

As you know, the way we do screenshots is in flux, so there's no hard and fast style guide at the moment. But I recommend you have a play with it, adding the drop shadow to the full page screenshots (and maybe trying it with some of the others too) to see what you think.

You'll notice that the CSS drop shadow does not exactly mimic the old pre-rendered drop shadow. That is a bug. I didn't have much time to work on this enhancement, so I settled on a close-ish approximation of our previous drop shadows.

I expect that in the future, we will make the CSS drop shadows look a little better (probably with some margin so they are not completely flush against the edges of their bounding box).

**Pull request merged**